erpnext/regional/report/hmrc_vat/hmrc_vat.py

```
@@ -0,0 +1,868 @@
+		return columns, data, message, chart, summary
+
+	def get_chart(self) -> dict:
+		"""Return chart data for the report."""
+		datasets = []
+		dataset_names = {3: "Output VAT", 4: "Input VAT"}
+		for box_id, box_name in dataset_names.items():
+			dataset = {
+				"name": box_name,
+			}
+			values = []
+			for _, _, label in self.periods:
```

Ruff (0.14.7)

Import `_` from line 9 shadowed by loop variable

(F402)